app/Services/Transcoding/SftpTranscodingStrategy.php

```
@@ -27,29 +27,28 @@ public function getTranscodeLocation(Song $song, int $bitRate): string
+                $bitRate,
+                File::hash($destination),
+                File::size($destination),
+            );
+        } finally {
+            File::delete($tmpSource);
+        }
 
         return $destination;
     }
 
     public function deleteTranscodeFile(string $location, SongStorageType $storageType): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$storageType'. (undefined)

(UnusedFormalParameter)